.travis.yml

```diff
@@ -11,9 +11,14 @@ matrix:
         - python: "3.7"
           dist: xenial
           sudo: true
+        - python: "3.8"
+          dist: xenial
+          sudo: true
 install:
+    - pip install -U pip
     - pip install coveralls coverage
     - pip install zope.testrunner
+    - pip install -U "zope.interface>=5.0.0"
```

Comment:
This looks weird?  Isn't zope.interface part of the regular requirements?  Would pip get the latest available version in any case?  Is this meant to get an early failure in case no release of zope.interface since 5.0.0 supports a particular Python version?

Reply:
This was an attempt for Travis Py 3.4 to get a more recent version than the old version that was apparently installed. I have dropped this change, since we drop 3.4.